src/PKSim.Core/Commands/NormalizeRelativeExpressionCommand.cs

```diff
@@ -57,7 +57,9 @@ protected override void PerformExecuteWith(IExecutionContext context)
          var allRelExpressionContainer = _molecule.AllExpressionsContainers();
          if (!allRelExpressionContainer.Any()) return;
          var max = allRelExpressionContainer.Select(x => x.RelativeExpression).Max();
-         allRelExpressionContainer.Each(relExp => relExp.RelativeExpressionNorm = max == 0 ? 0 : relExp.RelativeExpression / max);
-      }
+
+         //TODO ZTMSE
```

Comment:
@Yuri05 I have a bunch of TODOs everywhere that I'll start working on as I move through the tests.